src/aks-preview/azext_aks_preview/azuremonitormetrics/azuremonitorprofile.py

```diff
@@ -81,17 +81,22 @@ def ensure_azure_monitor_profile_prerequisites(
             cluster_name
         )
     else:
+        is_prometheus_enabled = False
         # Check if already onboarded
         if create_flow is False:
-            check_azuremonitormetrics_profile(cmd, cluster_subscription, cluster_resource_group_name, cluster_name)
-        # Do RP registrations if required
-        rp_registrations(cmd, cluster_subscription)
-        link_azure_monitor_profile_artifacts(
-            cmd,
-            cluster_subscription,
-            cluster_resource_group_name,
-            cluster_name,
-            cluster_region,
-            raw_parameters,
-            create_flow
-        )
+            is_prometheus_enabled = check_azuremonitormetrics_profile(cmd, cluster_subscription, cluster_resource_group_name, cluster_name)
+        if is_prometheus_enabled:
+            print("Azure Prometheus is already enabled : This command will only allow updates to the KSM parameters. All other parameters will be ignored")
```

Comment:
It is better to use a logger if possible.

Reply:
Updated to use logger.